handbook/engineering/roles.md

```diff
@@ -26,30 +27,56 @@ Some teams with more experienced members may need less guidance in certain areas
+  1. How does that work connect back to the team’s OKRs and roadmap?
+  1. For each OKR and roadmap item, is the team on track? The answer should be of the form:
+    - We expect to reach this goal.
+    - We are at risk of not reaching this goal because A, B, C.
+      - What can we change to get this back on track?
+    - We are not going to reach this goal because A, B, C.
+      - What did we learn from this failure to achieve our goals?
+  1. What did the team work on last week that wasn’t originally planned? Why?
+  1. What pain is the team experiencing?
 - Help keep the team focused on their goals while keeping them aware of the higher level picture.
 - Identify blockers or productivity issues and work with the team to address them.
 - Run weekly team sync meetings.
 - Communicate progress to stakeholders and other parts of the company (e.g. at company meeting).
 - Conduct monthly [retrospectives](../../retrospectives/index.md) with their team.
 - Scale themselves out by organizing and delegating work to other teammates.
 - Mentor and coach other teammates with timely feedback as appropriate in order to uphold team values and help them grow.
-- Collaborate with Engineering Managers to ensure team mates have everything they need to succeed and deliver on the goals of the team.
+- Collaborate with engineering managers to ensure teammates have everything they need to succeed and deliver on the goals of the team.
 
 ## Engineering Manager
 
-Engineering Managers are people managers who build teams of [Software Engineers](#software-engineer) to satisfy the needs of the business.
+Engineering managers lead, grow, and develop teams of software engineers.
 
 ### Responsibilities
 
-- Collaborate with [product](../product/index.md) to maintain a [prioritized roadmap](../../direction/index.md) for the work owned by your team.
-- Commit to a set of work that will be accomplished by the team for each [monthly release](releases/index.md) and capture those commitments in a [tracking issue](tracking_issues.md) on GitHub.
-- Conduct monthly [retrospectives](../../retrospectives/index.md) with your team.
-- Mentor and coach engineers through regular [1-1s](../leadership/1-1.md) to grow their careers.
-- [Review the performance](../people-ops/review-cycles.md) of your direct reports to determine compensation.
-- Interview candidates and own making improvements to our hiring process.
-- Review applications and make final hiring decisions for candidates interested in your team.
+Build an exceptional team that achieves results.
+
+- Ensure that the team has a roadmap that documents how it is going to achieve its [OKRs](../../company/okrs/index.md).
+- Ensure the the team's [tracking issue](tracking_issues.md) captures the work that each teammate has committed to for each [monthly release](releases/index.md).
+- Post a weekly update by EOD Monday on the tracking issue that answers the following questions:
```

Comment:
Is this redundant with the project lead section above? Could we extract it and have a Post weekly updates item that links to this list in one place?

Reply:
Yep, I moved this doc to the tracking issue page.